src/components/ContentEditor/WidgetEditor/WidgetEditor.tsx

```diff
@@ -64,20 +60,36 @@ export const WidgetEditor: React.FC<WidgetEditorProps> = ({ content, contentHash
 
         {editing && <>
             <div>
-                <FormControl className="content-editor-formcontrol">
-                    <InputLabel>Select a component type</InputLabel>
-                    <Select
-                        value={editedContent.type}
-                        onChange={(e) => {
-                            setEditedContentOnChange("type", e.target.value as string, editedContent, setEditedContent);
-                        }}>
-                        {Object.keys(WidgetTypes).map(widgetType => {
-                            return <MenuItem key={widgetType} value={widgetType}>
-                                {widgetType}
-                            </MenuItem>
-                        })}
-                    </Select>
-                </FormControl>
+                <div className="wiki-form">
+                    <form>
+                        <fieldset>
+                            <legend>Select a Widget</legend>
+                            <select id="widget" className="widget-type"
```

Comment:
using the id `widget` is a bit general. I would avoid that and instead have it be something like `widget-picker-select` or something.

Reply:
removed both id and className for the select tag because they were actually not being called in WidgetEditor.css